practice-two/src/components/FormInput/FormInput.css

```diff
@@ -0,0 +1,32 @@
+.form__input {
+  display: flex;
+  flex-direction: column;
+  width: 280px;
+}
+
+.input__field {
+  padding: 15px;
+  margin: 10px 0px;
+  border-radius: 5px;
+  border: 1px solid gray;
+}
+
+.input__label {
+  font-size: 12px;
+  color: gray;
+}
+
+.error__message {
+  font-size: 12px;
+  padding: 3px;
+  color: red;
+  display: none;
+}
+
+.input__field:invalid[focused='true'] {
+  border: 1px solid red;
+}
+
+.input__field:invalid[focused='true'] ~ .error__message {
+  display: block;
```

Comment:
why need set block style?

Reply:
Because my class .error__message is disable, it only appears when input is invalid.
I changed display: block to inline because I pay no attension that I use <span> tag

![image](https://github.com/datdragon777/React-Training/assets/74806826/2af1219e-d0e5-4ca1-942e-a6d65f40cb39)